src/lancer/utils.py

```diff
@@ -140,6 +140,18 @@ def isbuildin(name: str) -> bool:
     else:
         return False
 
+def isemptytype(type: int) -> bool:
+    """
+    Check whether type is an 'empty' token type.
+
+    Arguments:
+        type {int} -- The token type
+
+    Returns:
+        bool -- True if type is an 'empty' type, False otherwise.
+    """
+    return type in [NL, NEWLINE, INDENT, DEDENT]
```

Comment:
This has been moved from `variables.py`.

Note that `NEWLINE` has been added to the list of 'empty' types. This was necessary as some `NL`s from the initial `tokenize()` call seem to get converted to `NEWLINE`s after untokenizing and retokenizing with `untokenize()`/`tokenize()`.

This behaviour is not consistent with the explanation [here](https://stackoverflow.com/questions/27382508/what-is-the-difference-between-nl-and-newline-in-tokenizer-py), so I'm not really sure what's going on

Reply:
Cheers, thank you for your contribution! Maybe advertise your feature in the readme and we should be good to merge